internal/command/lint/command.go

```diff
@@ -89,7 +91,19 @@ func Command() *cobra.Command {
 			return errors.Errorf("found %d lint errors", len(result.Reports))
 		},
 	}
-	c.Flags().StringVar(&configPath, "config", "", "Path to config file")
+
+	c.Flags().String("config", "", "Path to config file")
 	c.Flags().BoolVarP(&verbose, "verbose", "v", false, "Enable verbose logging")
+	c.Flags().StringSlice("exclude", nil, "Exclude checks")
+	c.Flags().StringSlice("include", nil, "Include provided checks")
+	c.Flags().Bool("add-all-built-in", false, "Add All BuildIn checks")
+	c.Flags().Bool("no-defaults", false, "Don't add Default checks")
```

Comment:
We should make these names match the ones in the config (but convert camelCase to dashed-case).

The fact that we need to repeat the config parameters between Viper and the flags is not ideal. I think we can use code generation (similar to what we have done for our [templates](https://github.com/stackrox/kube-linter/blob/main/internal/templates/codegen/main.go)) where we generate code that can populate these flags automatically. That way, there is only one source of truth.

To be concrete, we should be able to write code that looks at:
```go
// ChecksConfig is the config that determines which checks to run.
type ChecksConfig struct {
	// AddAllBuiltIn, if set, adds all built-in checks. This allows users to
	// explicitly opt-out of checks that are not relevant using Exclude.
	AddAllBuiltIn bool `json:"addAllBuiltIn"`
	// DoNotAutoAddDefaults, if set, prevents the automatic addition of default checks.
	DoNotAutoAddDefaults bool `json:"doNotAutoAddDefaults"`
	// Exclude is a list of check names to exclude.
	Exclude []string `json:"exclude"`
	// Include is a list of check names to include. If a check is in both Include and Exclude,
	// Exclude wins.
	Include []string `json:"include"`
}

// Config represents the config file format.
type Config struct {
        // +viper=exclude
	CustomChecks []check.Check `json:"customChecks,omitempty"`
	Checks       ChecksConfig  `json:"checks,omitempty"`
}
```
walks the struct, relies on the comments for descriptions, and generates a function that can add flags for all paths that do NOT have a `viper=exclude` comment, like:
```
func AddFlags(c *cobra.Command, v *viper.Viper) {
    c.Flags().StringSlice("exclude", nil, "Exclude is a list of check names to exclude.")
     v.BindPFlag("checks.exclude", c.Flags().Lookup("exclude")
     // ... and so on
}
```

Reply:
thanks for the suggestion. I'll change it to suggested approach.